examples/middlewares/example_session.cpp

```diff
@@ -0,0 +1,146 @@
+#include "crow.h"
+#include "crow/middlewares/session.h"
+
+crow::response redirect()
+{
+    crow::response rsp;
+    rsp.redirect("/");
+    return rsp;
```

Comment:
generally a response name is shortened to `res`, I know it's not a big deal but IMO it's best to stay consistent.